### DISCUSSION TITLE
Maybe  ZED  can  add Python  venv slection  and  "Run "

### DISCUSSION THREAD

**Discussion by liu20001028** [zed-industries/zed#17627] [Feature Requests]
l think lt would be better if ZED could add  venv slection and  "Run code" . There are lots of people use lt   for data   and  learn program beginning

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
